Exiled.Events/Config.cs

```diff
@@ -51,7 +51,8 @@ public sealed class Config : IConfig
         /// <summary>
         /// Gets or sets a value indicating whether configs has to be reloaded every time a round restarts.
         /// </summary>
-        public bool ShouldReloadConfigsAtRoundRestart { get; set; }
+        [Description("Indicates whether configs has to be reloaded every time a round restarts")]
+        public bool ShouldReloadConfigsAtRoundRestart { get; set; } = false;
```

Comment:
There's no need to explicitly set it to false, it's already set to false by default if you don't specify it.

Reply:
other configs are well specify, so i think this should also be unified.